app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  attr_reader :token
+  has_many :creators
+  has_many :events
```

Comment:
**[REQUIRED]**
Let's call the user who created the event 'creator' as described ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from Odin)
We can do this by having 
```
has_many :events, foreign_key: :creator_id
```